visitors/http_rest/spec_visitor.go

```diff
@@ -347,7 +381,13 @@ func (*DefaultSpecVisitorImpl) EnterPrimitive(self interface{}, c SpecVisitorCon
 }
 
 func (*DefaultSpecVisitorImpl) VisitPrimitiveChildren(self interface{}, c SpecVisitorContext, vm VisitorManager, d *pb.Primitive) Cont {
-	return self.(DefaultSpecVisitor).VisitNodeChildren(self, c, vm, d)
+	if d != nil {
+		return visitStructMembers(c, vm, d,
+			"Value", d.Value,
+			"AkitaAnnotations", d.AkitaAnnotations, // TODO: don't recurse into this one?
```

Comment:
Given that we don't have a VisitAkitaAnnotations, should we not recurse into this value?

Reply:
We probably should have a `VisitAkitaAnnotations`. Maybe some day, when we auto-generate this code.